platforms/aws/giantnetes/variables.tf

```diff
@@ -210,16 +210,26 @@ variable "subnet_worker_1" {
 
 ### VPN ###
 
-variable "aws_customer_gateway_id" {
+variable "aws_customer_gateway_id_0" {
   description = "AWS customer gateway ID. Should be created manually. Leave blank to disable VPN setup and enable public access for bastions."
   default     = ""
 }
 
-variable "external_ipsec_subnet" {
+variable "aws_customer_gateway_id_1" {
+  description = "AWS customer gateway ID. Should be created manually. Leave blank to disable VPN setup and enable public access for bastions."
```

Comment:
nitpick, remove `Leave blank to disable VPN setup and enable public access for bastions.` as this is only true for `_0`